packages/cli/test/simulation/simulation.test.ts

```diff
@@ -1,213 +1,154 @@
+  inclusionDelayAssertions,
+  missedBlocksAssertions,
+  nodeAssertions,
+  nodeSyncedAssertions,
   syncCommitteeAssertions,
 } from "../utils/simulation/assertions.js";
-import {CLClient, CLParticipant, Job} from "../utils/simulation/interfaces.js";
-
-const nodeCases: {beaconNodes: number; validatorClients: number; validatorsPerClient: number}[] = [
-  {beaconNodes: 4, validatorClients: 1, validatorsPerClient: 32},
-];
-
-const forksCases: {
-  title: string;
-  params: {
-    altairEpoch: number;
-    bellatrixEpoch: number;
-    runTill: Epoch;
-  };
-}[] = [
-  {
-    title: "mixed forks",
-    params: {altairEpoch: 2, bellatrixEpoch: 4, runTill: 6},
-  },
-];
-
-for (const {beaconNodes, validatorClients, validatorsPerClient} of nodeCases) {
-  for (const {
-    title,
-    params: {altairEpoch, bellatrixEpoch, runTill},
-  } of forksCases) {
-    const testIdStr = [
-      `beaconNodes-${beaconNodes}`,
-      `validatorClients-${validatorClients}`,
-      `validatorsPerClient-${validatorsPerClient}`,
-      `altair-${altairEpoch}`,
-      `bellatrix-${bellatrixEpoch}`,
-    ].join("_");
-
-    // eslint-disable-next-line no-console
-    console.log(
-      JSON.stringify({
-        beaconNodes,
-        validatorClients,
-        validatorsPerClient,
-        altairEpoch,
-        bellatrixEpoch,
-      })
-    );
-    const env = new SimulationEnvironment({
-      beaconNodes,
-      validatorClients,
-      validatorsPerClient,
-      altairEpoch,
-      // TODO: Use extra delay until env.clock is based on absolute time
-      genesisSlotsDelay: SLOTS_PER_EPOCH * 2,
-      bellatrixEpoch,
-      logFilesDir: join(logFilesDir, testIdStr),
-    });
+import {CLClient, NodePairResult, ELClient} from "../utils/simulation/interfaces.js";
+import {SimulationEnvironment} from "../utils/simulation/SimulationEnvironment.js";
+import {getEstimatedTimeInSecForRun, logFilesDir, SIM_TESTS_SECONDS_PER_SLOT} from "../utils/simulation/utils.js";
+
+const genesisSlotsDelay = SLOTS_PER_EPOCH * 2;
+const runTillEpoch = 6;
+const syncWaitEpoch = 2;
+
+describe("simulation test - multi-fork", function () {
+  this.timeout(
+    `${getEstimatedTimeInSecForRun({
+      genesisSlotDelay: genesisSlotsDelay,
+      secondsPerSlot: SIM_TESTS_SECONDS_PER_SLOT,
+      runTill: runTillEpoch + syncWaitEpoch,
+      grace: 0.1, // 10% extra time
+    })}s`
+  );
+
+  const env = SimulationEnvironment.initWithDefaults(
+    {
+      id: "multi-fork",
+      logsDir: join(logFilesDir, "multi-fork"),
+      chainConfig: {
+        // eslint-disable-next-line @typescript-eslint/naming-convention
+        ALTAIR_FORK_EPOCH: 2,
+        // eslint-disable-next-line @typescript-eslint/naming-convention
+        BELLATRIX_FORK_EPOCH: 4,
+        // eslint-disable-next-line @typescript-eslint/naming-convention
+        GENESIS_DELAY: genesisSlotsDelay,
+        // eslint-disable-next-line @typescript-eslint/naming-convention
+        TERMINAL_TOTAL_DIFFICULTY: BigInt(10),
```

Comment:
Why difficulty 10? 

Also just add the top
```
/* eslint-disable @typescript-eslint/naming-convention */
```
to prevent having to repeat many times